### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by pclausen**
Fixing the bin/configlet lint to the best of my ability.

### CODE REVIEW

File: Testing/Temporary/CTestCostData.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+---
```
Comment (ErikSchierboom) [inline Testing/Temporary/CTestCostData.txt:1]: Maybe add the `Testing/Temporary` directory to the `.gitignore` file? Or are the files in this dir used?

File: config.json [hunk 2]
```diff
@@ -1,153 +1,217 @@
...
-        "loops",
-        "math"
-      ]
-    }
-
-  ]
-}
+  "slug": "fortran",
+  "version": 3,
+  "active": true,
```
Comment (ErikSchierboom) [inline config.json:5]: Is this track ready to become active? I've checked the [implemented exercises](https://github.com/exercism/fortran/tree/main/exercises/practice), and it looks like 11 exercises have been implemented. We usually require around 20 exercises to be implemented for a track to become active.

```suggestion
  "active": false,
```

File: config.json [hunk 3]
```diff
@@ -1,153 +1,217 @@
...
+    "concept_exercises": false,
+    "test_runner": false,
+    "representer": false,
+    "analyzer": false
+  },
+  "solution_pattern": "%{snake_slug}.f90",
+  "test_pattern": "%{snake_slug}_test.f90",
+  "test_runner": {
+    "average_run_time": 3.0
+  },
```
Comment (ErikSchierboom) [inline config.json:17]: This track does not currently have a test runner (see https://github.com/exercism/fortran-test-runner), so this key should be removed. See [the docs](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#metadata).

File: config.json [hunk 4]
```diff
@@ -1,153 +1,217 @@
...
+  "active": true,
+  "blurb": "Fortran is one of the oldest languages still in use today, latest revision is 2018. The name Fortran is derived from FORmula TRANslation. It still shines for numerical high performance computing.",
+  "status": {
+    "concept_exercises": false,
+    "test_runner": false,
+    "representer": false,
+    "analyzer": false
+  },
+  "solution_pattern": "%{snake_slug}.f90",
+  "test_pattern": "%{snake_slug}_test.f90",
```
Comment (ErikSchierboom) [inline config.json:14]: These properties are deprecated and should be replaced by our new `files` property. See [the docs](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#files).

File: config.json [hunk 5]
```diff
@@ -1,153 +1,217 @@
...
+  "exercises": {
+    "concept": [],
+    "practice": [
+      {
+        "slug": "hello-world",
+        "name": "Hello World",
+        "uuid": "b71d9258-78ef-4986-b9ed-ebf93b72d399",
+        "core": false,
+        "auto_approve": true,
+        "unlocked_by": null,
```
Comment (ErikSchierboom) [inline config.json:45]: These properties are no longer used in Exercism v3, and can be removed from all exercises. See [the docs](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises).

**Review by ErikSchierboom — COMMENTED** [review]
Looking good! I've left some small comments.

File: Testing/Temporary/CTestCostData.txt (hunk 1 above)
Comment (pclausen) [reply Testing/Temporary/CTestCostData.txt:1]: no, wrong git add from my side. Let me check

File: config.json (hunk 4 above)
Comment (pclausen) [reply config.json]: Removed in last commit

File: config.json (hunk 3 above)
Comment (pclausen) [reply config.json]: Removed in last commit

File: Testing/Temporary/CTestCostData.txt (hunk 1 above)
Comment (pclausen) [reply Testing/Temporary/CTestCostData.txt]: Removed in last commit
@ErikSchierboom Thank you for your review and sorry for not reading the docs in enough detail.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by kytrinyx** [comment]
@pclausen This PR adds a Python script `bin/fetch-configlet.py` (https://github.com/exercism/fortran/pull/107/files#diff-e6e97beebc5d27cadf01cc831377ba2c1d4652161714e9c4ab2377553756ae6d) -- do you happen to recall why you added it? Could it have been an accidental addition?

**Pull request merged**